assets/javascripts/kitten/components/modals/next/index.js

```diff
@@ -147,13 +147,17 @@ const ModalTitle = ({ children }) => (
   </Title>
 )
 
-const ModalParagraph = ({ children, align }) => (
+const ModalParagraph = ({ children, withoutMargin, className, align }) => (
   <StyledParagraph
     modifier="quaternary"
-    margin={false}
     style={{ textAlign: align }}
     tag="p"
-    className="k-u-margin-bottom-triple k-u-margin-bottom-quadruple@s-up"
+    className={classNames('k-Modal__paragraph', className,
+      'k-u-margin-bottom-triple',
+      'k-u-margin-bottom-quadruple@s-up',
+      'k-u-margin-top-none',
+      {'k-u-margin-bottom-none--important': withoutMargin},
+    )}
```

Comment:
Pour éviter le `--important`, est-ce qu'on utiliserait pas la condition pour chaque classe ?

```suggestion
    className={classNames('k-Modal__paragraph', className,
      'k-u-margin-top-none',
      {
        'k-u-margin-bottom-triple': !withoutMargin,
        'k-u-margin-bottom-quadruple@s-up': !withoutMargin,
      },
    )}
```